src/identity/apis/auth.ts

```diff
@@ -313,3 +315,38 @@ export const getDefaultAccountDefaultOrg = async (): Promise<
     throw new GenericError('No default account found')
   }
 }
+
+export const getUserAccounts = async (): Promise<UserAccount[]> => {
+  const response = await getAccounts({})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+  if (!Array.isArray(response.data)) {
+    throw new GenericError('No account found')
+  }
+
+  return response.data
+}
+
+export const updateUserAccount = async (accountId, name) => {
+  const response = await patchAccount({accountId, data: {name}})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+  if (response.status === 422) {
+    throw new UnprocessableEntityError(response.data.message)
+  }
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+  if (response.status !== 200) {
```

Comment:
What's the rationale for this one - do we expect anything other than a `200`, `422`, or `500`? This does look more cautious, but I only see `200`, `422`, and the `default` server error in the openapi spec.

Reply:
yea, being extra cautious. makes sense to remove it. will do.

Reply:
Paranoia is good in my book - I think it's okay if you want to leave it in.